src/helpers/file_helpers.sh

```diff
@@ -3,15 +3,6 @@
 # Helper methods for dealing with files and temp files.
 
 
-# Unique string that identifies the current directory
-temp_filename_suffix="$(pwd | tr '/' '_')"
```

Comment:
These changes were necessary here to make the specs pass, so I couldn't extract them into a separate PR.